make-astyle.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+curl -L http://downloads.sourceforge.net/project/astyle/astyle/astyle%202.05.1/astyle_2.05.1_linux.tar.gz\
```

Comment:
If you add a check here first for `usr/local/bin/astyle` then you won't need to run this every time.